web/api/mastermind/order.js

```diff
@@ -0,0 +1,72 @@
+const Joi = require('joi');
+const auth = require('../../lib/authentications');
+const validation = require('../validations/orders');
+const orderHandlers = require('../../lib/handlers/order');
+
+const basePath = '/api/3.0';
+const { applications } = validation.definitions;
+
+module.exports = [
+  {
+    method: 'GET',
+    path: `${basePath}/events/{eventId}/orders/{orderId}`,
+    handler: orderHandlers.get(),
+    config: {
+      auth: auth.apiUser,
+      description: 'List A User\'s Order',
```

Comment:
*load, not list :)
Alos, it's the current user, not "A" user, as you only load the current user info